src/components/ControlledTimePicker.tsx

```diff
@@ -1,63 +1,51 @@
 import { TimePicker } from "antd";
-import { Dayjs } from "dayjs";
+import dayjs, { Dayjs } from "dayjs";
+import utc from "dayjs/plugin/utc";
+dayjs.extend(utc);
 import { ReactNode, useEffect, useState } from "react";
 import { Control, Controller } from "react-hook-form";
 import { OnboardingFormInputs } from "../pages/profile";
-import { ErrorDisplay } from "../styles/profile";
-
-interface ControlledTimePickerProps {
+interface ControlledTimePickerRHFProps {
   control: Control<OnboardingFormInputs>;
   name: "startTime" | "endTime";
   placeholder?: string;
   value?: Date;
 }
-
-const ControlledTimePicker = (props: ControlledTimePickerProps) => {
-  const dayjs = require("dayjs");
-  const utc = require("dayjs/plugin/utc");
-  dayjs.extend(utc);
-
-  const [displayedTime, setDisplayedTime] = useState<Dayjs | null>(null);
+const ControlledTimePickerRHF = (props: ControlledTimePickerRHFProps) => {
+  const [displayedTime, setDisplayedTime] = useState<Dayjs | undefined>(
+    props.value ? dayjs.utc(props.value) : undefined
+  );
 
   useEffect(() => {
     if (props.value) {
-      setDisplayedTime(dayjs.utc(props.value));
+      const utcOffset = props.value.getTimezoneOffset();
+      setDisplayedTime(dayjs(props.value).add(utcOffset, "minute"));
```

Comment:
Would it not be possible to do the same thing as line 16 here?

Reply:
yes idk why i did it this way previously. just changed in new commit.